src/test/java/net/minecraftforge/debug/EnchantmentLevelEventTest.java

```diff
@@ -0,0 +1,30 @@
+package net.minecraftforge.debug;
+
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.living.LivingEnchantmentLevelEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+@Mod(modid="enchantmentleveleventtest",version="1.0")
+public class EnchantmentLevelEventTest
+{
+
+    @Mod.EventHandler
+    public void preinit(FMLPreInitializationEvent evt)
+    {
+        MinecraftForge.EVENT_BUS.register(this);
+    }
+
+    @SubscribeEvent
+    public void onInteract(LivingEnchantmentLevelEvent event)
+    {
+        if (event.getEntityLiving().isSneaking())
+        {
+            int oldLevel = event.getLevel();
+            int newLevel = (oldLevel + 1) * 10;
```

Comment:
This code doesn't multiply by 10 as described. If the player already has level 1 of an enchantment the result would be 20 instead of 10. You should try to make the test mod match the description as closely as possible so reviewers don't get confused.